blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitReadSaveService.java

```diff
@@ -68,6 +69,7 @@
 public class GitReadSaveService extends ScmContentProvider {
 
     private static final Logger logger = LoggerFactory.getLogger(GitReadSaveService.class);
+    @SuppressFBWarnings(value={"NP_NONNULL_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR"}, justification="https://github.com/spotbugs/spotbugs/issues/2044")
```

Comment:
not really sure if this is the issue. It seems like if it really was the issue it wouldn't have been ignored for so long...most likely something i need to fix (environment?). Wanted to quickly get past this error for now to do more testing.

Reply:
https://github.com/spotbugs/spotbugs/issues/2044 FTR

```suggestion
    @SuppressFBWarnings(value={"NP_NONNULL_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR"}, justification="https://github.com/spotbugs/spotbugs/issues/2044 TYPE is mutated by setType")
```

Reply:
resolved in e4f44aba58fa7baf959f7a5b30aad54f599c566e